pkg/api/handlers/compat/containers.go

```diff
@@ -289,7 +289,7 @@ func LibpodToContainer(l *libpod.Container, sz bool) (*handlers.Container, error
 		return nil, err
 	}
 	stateStr := state.String()
-	if stateStr == "configured" {
+	if stateStr == "initialized" {
 		stateStr = "created"
 	}
```

Comment:
I think we should match the const values here, this will make sure we catch this again if someone renames the values.

Reply:
I can make this change in this PR. But I assume anyone can change the `String()` method for the enums to return an unknown Docker value for any of the other variants and forget about this again.

Maybe tests will be better? Are there any test examples for the compat layer I can look at?

Reply:
Most of our APIv2 testing lives in test/apiv2/ and intermingles testing for both the Libpod and Compat APIs; that's probably the best place

Reply:
What I mean is the you should check if this is `ContainerStateCreated`, in this case it does not matter when the underlying sting is changed.

Reply:
I added some tests and couldn't understand why they kept failing... Seems like I needed to update the JSON function too :smile: